stories/timeline/props-table.tsx

```diff
@@ -123,33 +123,10 @@ const DATA: ApiTableSectionProps[] = [
                 mandatory: true,
                 description: (
                     <>
-                        The color style of the {code("Pill")}. <br />
-                        <b>
-                            Note: from v2.4.0 onwards, values {code(`"dark"`)}{" "}
-                            and {code(`"light"`)}
-                            will no longer be supported in favour of{" "}
-                            {code(`"solid"`)} and
-                            {code(`"outline"`)} respectively.
-                        </b>
+                        The type style of the {code("Pill")}. <br />
```

Comment:
Can add link to the `Pill` story

Reply:
Sorry actually i realised to just add the link to the Pill story on the `PillProps` at line 80. Since the props are exactly the same, there is no need to copy them. 
Can use this link
https://designsystem.life.gov.sg/react/index.html?path=/docs/modules-pill--docs#component-api